.gitignore

```diff
@@ -10,6 +10,8 @@ __pycache__
 output/*.xml
 .coverage
 coverage.xml
+.netrc
```

Comment:
To all,
I purposely added an empty .netrc to this PR because otherwise we get a weird error netrc is a directory and the build fails https://jenkins.conjur.net/blue/organizations/jenkins/cyberark--conjur-api-python3/detail/add_init_logic/48/pipeline#step-55-log-51

Reply:
I can't see that log but this may mean that we need sturdier tests maybe instead.

Reply:
Tests on the Jenkins side? or on our side

Reply:
Could be both but likely in the test